lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -798,6 +797,40 @@ def _histogram_binding_profile(self, n_time_bins, bandwidth, n_position_points):
 
         return x.squeeze(), densities
 
-    @use_docstring_from(KymoTrack.estimate_diffusion)
-    def estimate_diffusion(self, *args, **kwargs):
-        return [k.estimate_diffusion(*args, **kwargs) for k in self._src]
+    def estimate_diffusion(self, method, *args, min_length=None, **kwargs):
+        r"""Estimate diffusion constant for each track in the group.
+
+        Calls :meth:`KymoTrack.estimate_diffusion` for each track. See the documentation for that
+        method for more detailed information and references.
+
+        Parameters
+        ----------
+        method : {"cve", "ols", "gls"}
+            - "cve" : Covariance based estimator [5]_. Optimal if SNR > 1. Can only be used when
```

Comment:
Should either add the references or remove the `[5]_` etc because this will lead to errors in the docs build.